Hosting API Documentation.md

```diff
@@ -0,0 +1,112 @@
+# Summary
+
+This is the flow we are using to generate and host our documentation:
+
+- Write tests using rspec and annotate them using [dox](https://github.com/infinum/dox)
+- Once tests on [semaphore](https://semaphoreci.com) pass we move to deploy
+- In the deploy phase we do the following:
+  * deploy the application using [mina](https://github.com/mina-deploy/mina)
+  * run only dox tests to generate the documentation
+  * copy the generated documentation to the application server inside `public` folder
+
+# Why
+
+We settled on serving our documentation for the application server because it is convenient, easiest
+and free.
+
+Solutions like apiary.io require us to have one project for each environemnt, can't easily set
+a desired domain and can't easily hide behind some kind of authentication (like Basic Authentication).
+
+Copying after a successful deploy we assure that the documentation is always up to date.
```

Comment:
I think `ensure` would be a better fit here instead of "assure". @cukoreva confirm pls :)

Reply:
Yes, `ensure` would be a better solution

Reply:
Also, `By copying after a successful deploy, we ensure`